src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py

```diff
@@ -8440,7 +8440,7 @@ def test_aks_update_with_azurekeyvaultkms_public_key_vault(self, resource_group,
 
     @live_only()
     @AllowLargeResponse()
-    @AKSCustomResourceGroupPreparer(random_name_length=17, name_prefix='clitest', location='centraluseuap', preserve_default_location=True)
+    @AKSCustomResourceGroupPreparer(random_name_length=17, name_prefix='clitest', location='westcentralus', preserve_default_location=True)
```

Comment:
please also remove `preserve_default_location=True`

Reply:
what about the following cases?
- test_aks_update_with_azurekeyvaultkms_private_key_vault
- test_aks_create_with_azurekeyvaultkms_private_cluster_v1_private_key_vault

Could you please help update the cases in aks-preview?

Reply:
fixed